tests/test_textloader.py

```diff
@@ -122,9 +164,24 @@ def test__read_text_parquet(mock_read_parquet):
     actual_result = dummy_instance._read_text_parquet(files_path)
 
     # Then
-    mock_read_parquet.assert_called_once_with(files_path)
+    mock_read_parquet.assert_called_once_with(files_path, encoding=encoding)
     assert_frame_equal(expected_result.compute(), actual_result.compute())
 
+@patch("nlpretext.textloader.pandasloader.read_parquet")
+def test__read_text_parquet_pandas(mock_read):
+    # Given
+    files_path = "some_path/to_read.txt"
+    file_format = "txt"
+    encoding = "utf-8"
+    text_column = "text"
+
+    dummy_instance = TextLoader(file_format=file_format, use_dask=False, encoding=encoding, text_column=text_column)
+    dummy_instance._read_text_parquet(files_path)
+
+    # Then
+    mock_read.assert_called_once_with(files_path, encoding=encoding)
+
+
 
 @pytest.mark.parametrize(
```

Comment:
Could be nice to add tests examples with use_dask = True and use_dask = False, to make sure everything works properly in both cases

Reply:
Yes but as we mock the real functions, and we don't have end to end test.  there is no way to really test it

Reply:
I may be wrong but I think that if you specify use_dask when calling the dummy textloader, it will call a mocked function or another depending on the value, whereas currently it only calls dask functions (since use_dask = True is the default value)